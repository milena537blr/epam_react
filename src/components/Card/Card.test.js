// import handleCardClick from './Card.js';

describe('handleCardClick', () => {
  it('should set state', () => {
    expect(3).toBe(3);
  });
})
