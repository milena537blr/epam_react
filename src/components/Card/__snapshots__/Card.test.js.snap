// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card snapshot test 1`] = `
<Card
  card={
    Object {
      "date": 2007,
      "genre": "Dramas",
      "id": 1,
      "name": "Vampire diaries",
      "time": 145,
    }
  }
>
  <figure
    className="card"
  >
    <button
      area-label="Vampire diaries"
      onClick={[Function]}
    >
      <img
        alt="Vampire diaries"
        className="image"
        src="test-file-stub"
      />
    </button>
    <figcaption>
      <Box
        align="space-between"
        direction="horizontal"
        inline={false}
        verticalAlign="middle"
      >
        <div
          className="root horizontal spaceBetween middle"
          style={
            Object {
              "height": "undefined",
              "margin": "undefined",
              "marginBottom": "undefined",
              "marginLeft": "undefined",
              "marginRight": "undefined",
              "marginTop": "undefined",
              "maxHeight": "undefined",
              "maxWidth": "undefined",
              "minHeight": "undefined",
              "minWidth": "undefined",
              "padding": "undefined",
              "paddingBottom": "undefined",
              "paddingLeft": "undefined",
              "paddingRight": "undefined",
              "paddingTop": "undefined",
              "width": "undefined",
            }
          }
        >
          <div
            className="name"
          >
            Vampire diaries
          </div>
          <div
            className="date"
          >
            2007
          </div>
        </div>
      </Box>
      <div
        className="genre"
      >
        Dramas
      </div>
    </figcaption>
  </figure>
</Card>
`;
